baselines/dom.generated.d.ts

```diff
@@ -17035,7 +17040,6 @@ declare var Image: {
 declare var Option: {
     new(text?: string, value?: string, defaultSelected?: boolean, selected?: boolean): HTMLOptionElement;
 };
-declare var HTMLDocument: Document;
```

Comment:
Ooh, I’m guessing this was an incorrect attempt at back compat, right? So until now the `HTMLDocument` global value appeared to actually be an _instance_ of an `HTMLDocument`?

Reply:
Not quite, the general pattern for a lot of DOM types is a `var` with an `interface` - as that's globally extendable and generally represents the runtimey flexiness of JS. This still represented the 'classlike' ness if you will, kind surprised that we didn't get a constructor though

Reply:
That’s not what this says though... this var doesn’t have a constructor-ish type like the one added back by the PR, it has an instance-ish type. There’s no class-likeness here, which is why ant-design had an error.

Reply:
I think I’d be ok with that approach, but personally don’t see a strong reason to remove `HTMLDocument` (or even to mark it deprecated)—I must be missing the original context.

Reply:
`HTMLDocument` is not defined as a separate interface anymore by the HTML spec and served as a legacy alias of `Document`. But since both Firefox and Chrome haven't implemented things as the spec says, maybe this also make sense as-is.